airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```
@@ -97,31 +97,32 @@
               "type": "string",
               "multiline": true,
               "title": "Custom Query",
-              "description": "A custom defined GAQL query for building the report. Should not contain segments.date expression because it is used by incremental streams. See Google's <a href=\"https://developers.google.com/google-ads/api/fields/v11/overview_query_builder\">query builder</a> for more information.",
+              "description": "A custom defined GAQL query for building the report. Avoid including the segments.date expression as it is used by incremental streams. For more information, refer to <a href=\"https://developers.google.com/google-ads/api/fields/v11/overview_query_builder\">Google's documentation</a>.",
               "examples": [
                 "SELECT segments.ad_destination_type, campaign.advertising_channel_sub_type FROM campaign WHERE campaign.status = 'PAUSED'"
               ]
             },
             "table_name": {
               "type": "string",
               "title": "Destination Table Name",
-              "description": "The table name in your destination database for choosen query."
+              "description": "The table name in your destination database for the chosen query."
             }
           }
         }
       },
       "login_customer_id": {
         "type": "string",
         "title": "Login Customer ID for Managed Accounts",
-        "description": "If your access to the customer account is through a manager account, this field is required and must be set to the customer ID of the manager account (10-digit number without dashes). More information about this field you can see <a href=\"https://developers.google.com/google-ads/api/docs/concepts/call-structure#cid\">here</a>",
+        "description": "If your access to the customer account is through a manager account, this field is required, and must be set to the 10-digit customer ID of the manager account. For more information about this field, refer to <a href=\"https://developers.google.com/google-ads/api/docs/concepts/call-structure#cid\">Google's documentation</a>.",
+        "pattern_descriptor": "The customer ID must be 10 digits, with no dashes.",
```

the UI already adds a prefix to pattern_descriptors https://docs.airbyte.com/connector-development/connector-specification-reference/#pattern-descriptors so this should just describe the pattern
```suggestion
        "pattern_descriptor": ": 10 digits, with no dashes.",
```